x-pack/plugins/security_solution/public/timelines/components/graph_overlay/index.tsx

```diff
@@ -137,6 +139,17 @@ const GraphOverlayComponent = ({
     globalFullScreen,
   ]);
 
+  const { signalIndexName } = useSignalIndex();
+  const [siemDefaultIndices] = useUiSetting$<string[]>(DEFAULT_INDEX_KEY);
+  const indices: string[] = useMemo(() => {
```

Comment:
Concat the signal indices and the SIEM default indices and pass that to Resolver.